clinics/clinics.go

```diff
@@ -153,6 +154,11 @@ type MRNSettings struct {
 	Unique   bool `bson:"unique"`
 }
 
+type TagsSettings struct {
```

Comment:
`EHRTagsSettings` in `types.go` is automatically generated. Persisting models generated from the API spec is a recipe for disaster, because the code generator often changes its output.